sktime/classification/shapelet_based/tests/test_stc.py

```diff
@@ -0,0 +1,34 @@
+
+# import numpy as np
+# from sklearn.model_selection import train_test_split
+# from sktime.classification.shapelet_based import ShapeletTransformClassifier
+# from sktime.datasets import load_italy_power_demand
+
+
+# def test_stc_with_pd():
```

Comment:
please remove the file if no longer needed - if we want to support y as a np.array we should probably add this to our collection of estimator checks and run it on all